### PR TITLE
Reporting tables

### PR DESCRIPTION
**Pull request by EverardoAlcarazBaltazar**
# Description

What did you implemented/Why did you implemented this:

We implement a report table for stores and employees because it is necessary to show specific information to admins.

## Screenshots

Screenshot from 2023-02-03 13-38-23
Screenshot from 2023-02-03 13-38-37

### CODE REVIEW

File: app/assets/stylesheets/main.css.scss [hunk 1]
```diff
@@ -8,3 +8,4 @@
 @import './pages/crud_store';
 @import './pages/login_admin';
 @import './pages/register_forms';
+@import './pages/report_form'
```
Comment (juperameza) [inline app/assets/stylesheets/main.css.scss:11]: Add a new line at the end of this file

File: app/assets/stylesheets/pages/_report_form.css.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.report-button{
+  width: 168px;
+  height: 77px;
+  left: 513px;
+  top: 284px;
+  background: #7B7B7B;
+  margin-bottom: 20px;
```
Comment (juperameza) [inline app/assets/stylesheets/pages/_report_form.css.scss:7]: Alphabetically order this file

File: app/assets/stylesheets/pages/_report_form.css.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    color: #FFFFFF;
+}
+.button-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    
+
+}
```
Comment (juperameza) [inline app/assets/stylesheets/pages/_report_form.css.scss:32]: Add a new line at the end of this file

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by oscaregr — APPROVED** [review]

File: app/views/reports/employee.html.erb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          <li class="employees-table-cell">
+            <p><%= check.check %></p>
+          </li>
+      
+        </ul>
+      </section>
+    <%end%>
+  </section>
+</div>
+</div>
```
Comment (oscaregr) [inline app/views/reports/employee.html.erb:53]:
```suggestion
</div>

```

add one line  here

File: app/views/reports/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+            <div>
+              <button class="button-edit"> <%= link_to "Details" , report_path(store) %></button>
+            </div>
+          </li>
+        </ul>
+      </section>
+    <%end%>
+  </section>
+</div>
+</div>
```
Comment (oscaregr) [inline app/views/reports/index.html.erb:53]:
```suggestion
</div>

```

add one line here

**Review by oscaregr — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]
LGTM

**Review by oscaregr — APPROVED** [review]
LGTM

**Review by ikerismak — COMMENTED** [review]
all reviewed

**Review by ikerismak — APPROVED** [review]

**Pull request merged**